test/benchmark-common-tasks/main.cpp

```diff
@@ -1630,6 +1630,72 @@ struct BenchmarkSort : BenchmarkWithStrings {
     }
 };
 
+// benchmark for testing compressed string sorting.
+// N is the size of the string to generate
+// M is the numer of times we want store the string (number of dups)
+template <size_t N, size_t M>
+struct BenchmarkSortCompressed : BenchmarkWithStrings {
+    mutable std::string _name;
```

Comment:
why is this mutable? the leading underscore is also unusual for our naming conventions.